README.md

```diff
@@ -75,32 +75,55 @@ See <a href="https://semgrep.dev/docs/supported-languages/">supported languages<
 
 ### Getting started
 
-To install Semgrep use Homebrew or pip, or run without installation via Docker:
+1.  Register to <a href="http://bit.ly/424Jbs0" target="_blank">semgrep.dev</a>
+<img src="images/semgrep-main-image.jpg" /> 
 
-```sh
-# For macOS
-$ brew install semgrep
 
-# For Ubuntu/WSL/Linux/macOS
-$ python3 -m pip install semgrep
+2.  Install Semgrep CLI
 
-# To try Semgrep without installation run via Docker
-$ docker run --rm -v "${PWD}:/src" returntocorp/semgrep semgrep
-```
+  ```
+  # For macOS
+  $ brew install semgrep
 
-Once installed, Semgrep can run with single rules or entire rulesets. Visit [Docs > Running rules](https://semgrep.dev/docs/running-rules/) to learn more or try the following:
+  # For Ubuntu/WSL/Linux/macOS
+  $ python3 -m pip install semgrep
 
-```sh
-# Check for Python == where the left and right hand sides are the same (often a bug)
-$ semgrep -e '$X == $X' --lang=py path/to/src
+  # To try Semgrep without installation run via Docker
+  $ docker run --rm -v "${PWD}:/src" returntocorp/semgrep semgrep
+  ```
+
+3.  Run `semgrep login` to get the login URL. Open the login URL in the browser and login.
+
+  ```
+  semgrep login
+  ```
+<img src="images/semgrep-login.jpg" /> 
+
+4. Go to your app's root directory and run the following:
 
-# Fetch rules automatically by setting the `--config auto` flag.
-# This will fetch rules relevant to your project from Semgrep Registry.
-# Your source code is not uploaded.
-$ semgrep --config auto
 ```
+semgrep --config auto
+```
+<img src="images/semgrep-auto-config-example.jpg" /> 
+
+### Set up CI/CD in two minutes
```

Comment:
My suggestion is to keep the existing system, but add a new block about Semgrep.dev login and connected scans. This matches what I see in sibling tool READMEs, allowing us to highlight the value of going deeper into the Semgrep ecosystem.